### ISSUE TITLE
[Sev1] List items under the "Menu" button in the new panel are not keyboard accessible

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#116514]
**Steps to reproduce**:
1. Create a new dashboard
2. Navigate to "Add visualization" button and hit ENTER key. "Select data source" popup will appear. 
3. Navigate to the TestData list item select it. 
4. Now navigate to the tool bar and select back to dashboard control and select. 
5. Navigate to the add menu present in tool bar and select add row control. 
6. Navigate to the  view ,edit, More , Remove  list items present in the New panel more action menu under row title. 
7. Verify whether the list items in the menu button under “New Pane” tab are keyboard accessible or not. 

**Actual Result**:
The list items under the "Menu" button cannot be activated using keyboard inputs such as Space or Enter. Users must rely on mouse clicks to perform the desired actions. 
Observation: 
The list items under the "Menu" button in the new panel are not operable via keyboard. Specifically, pressing the Space or Enter keys does not activate the list items, which are only activated by mouse clicks. This makes the functionality inaccessible for keyboard users. 

Issue1: Same issue repro for the expand/collapse buttons after activating the refresh control under the Query tab. 
Navigation: Select “ Add visualization” button > Select testdata list item> Select Last 7 days in the combo box above new panel> Activate Query inspector in the Queries tab > Activate refresh button in the query tab> Observe that keyboard focus doesnot move to the expand/ collapse buttons after the button controls. 

Issue2: Same issue repro for the expand/collapse buttons for objects under the Add action tab. 
Navigation: Select “ Add visualization” button > Select testdata list item> Select Last 7 days in the combo box above new panel> Activate Query inspector in the Queries tab > Activate refresh button in the query tab> Observe that keyboard focus doesnot move to the expand/ collapse buttons after the button controls. 

**Expected Result**
The list items under the "Menu" button should be fully operable using keyboard navigation. Users should be able to: 
Navigate to the list items using the Tab key. 
Activate the list items by pressing the Space or Enter keys. 

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
WCAG: [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)
Section 508: [E207.2 WCAG Conformance](https://www.access-board.gov/ict/#E207.2), [E205.4 Accessibility Standard](https://www.access-board.gov/ict/#E205.4) and [E205.4.1 Word Substitution when Applying WCAG to Non-Web Documents](https://www.access-board.gov/ict/#E205.4.1), and [E207.2.1 Word Substitution when Applying WCAG to Non-Web Software](https://www.access-board.gov/ict/#e207-2-1-word-substitution-when-applying-wcag-to-non-web-softwar).
EN 301 549: [PDF of EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), which applies WCAG principles for Keyboard to 9.2.1.1 web, 10.2.1.1 non-web documents, and 11.2.1.1.1 software.

**Environment**:
- Grafana version: 12.2.0
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by JonCole** [comment]
We consider this a Sev 1 issue and ask that it be fixed ASAP

**Comment by DivyamUp14** [comment]
@JonCole I rechecked the keyboard behavior for the menu:

- After the menu opens, focus moves to the first item
- Menu items are reachable using Arrow Keys
- Items can be activated using Enter

With this behavior, the menu is operable via keyboard.

### Clarification Needed

Can you confirm whether the expectation is that **menu items should also be reachable using the Tab key**, instead of arrow-key navigation?

Want to confirm the expected interaction before changing the behavior.

**Comment by JonCole** [comment]
@DivyamUp14 I looked through the WAG requirement, but I wasn't sure, so I have asked for clarification from our Accessibility team.  I will get back with you as soon as possible.

**Comment by JonCole** [comment]
Our accessibility test team is no longer able to reproduce this issue - so we can close it out.
